go/test/endtoend/cluster/cluster_process.go

```diff
@@ -299,6 +300,69 @@ func (cluster *LocalProcessCluster) StartKeyspace(keyspace Keyspace, shardNames
 	return
 }
 
+// LaunchCluster creates the skeleton of a cluster. The user then have to start all the services.
+// Does not start Mysqlctl process or vttablet process
+// No database creation
+// No Init shard master
+// Does not Apply Schema or Vschema
```

Comment:
what does the skeleton contain? The way to start services is by calling methods like `StartVtgate`? Please change the comments to make this clear.